src/Resources/contao/library/Contao/Encryption.php

```diff
@@ -157,7 +166,12 @@ protected static function initialize()
 			throw new \Exception('Error initializing encryption module');
 		}
 
-		if (\Config::get('encryptionKey') == '')
+		/** @var KernelInterface $kernel */
+		global $kernel;
+
+		$container = $kernel->getContainer();
+
+		if ($container->getParameter('framework.secret') == '')
```

Comment:
Yeah, the check would already throw an exception now. If you wanted to keep it you'd need to check for `->has()` :)